<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CounterAnnotationFormatWidget</class>
 <widget class="QWidget" name="CounterAnnotationFormatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counter Format</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="l1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Select Counter Format</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="pTimeBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QLabel" name="l2">
        <property name="text">
         <string>Field Width</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="pFieldWidth">
        <property name="toolTip">
         <string>Set Field Width</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="pPadding">
        <property name="toolTip">
         <string>Pad with Zeros</string>
        </property>
        <property name="text">
         <string>Pad with zeros</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pReset">
        <property name="toolTip">
         <string>Reset Counter</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
